### PR TITLE
Use of environment variables

### LINKED ISSUE #2: Use a .env file for usernames and passwords
Load the **username and password** from an `.env ` file instead of hardcoding it in the python code.
Never hardcode password or usernames in the code - Best Practice.

### PR DESCRIPTION
**Pull request by sighclone**
Hi! this is for issue #2. I have created a .env file and appropriately use `f'{config["username"]}'` wherever appropriate.<br>
This is my first time doing a no "first time" label issue. Please do check the code once as I am afraid I might have made some mistake.

### CODE REVIEW

File: subscriber.py [hunk 1]
```diff
@@ -23,6 +24,9 @@ def on_disconnect(client, userdata, rc):
     if rc != 0:
         print("Unexpected disconnection.")
 
+# load the .env values
+config = dotenv_values(".env")
```
Comment (Santhoshkumard11) [inline subscriber.py]: @sighclone guess you can move the loading the .env file to the import section of the code as you have done in the published and make sure you have a good commit message - not just update publisher.py 

It's should be something like **updated pusblisher.py -  to import .env file and use the configs for usernames and passwords**

**Review by Santhoshkumard11 — CHANGES_REQUESTED** [review]

File: subscriber.py (hunk 1 above)
Comment (sighclone) [reply subscriber.py]: Oh okay, I'll do it right now!

**Review by Santhoshkumard11 — COMMENTED** [review]
looking good now

**Comment by sighclone** [comment]
Yupp I've made the changes

**Review by Santhoshkumard11 — COMMENTED** [review]
Thanks for making the change.

**Review by Santhoshkumard11 — APPROVED** [review]

**Pull request merged**